docs/_themes/clusterhq/layout_homepage.html

```diff
@@ -127,10 +127,11 @@ <h2 class="heading text-center doc-banner">See our <a href="https://docs.cluster
 				<span class="widget-title">Install Flocker on:</span>			
 				<div class="textwidget">
 				<ul>
-					<li><a href="https://docs.clusterhq.com/en/latest/gettingstarted/installation.html#using-amazon-web-services">AWS</a></li>
-					<li><a href="https://docs.clusterhq.com/en/latest/gettingstarted/installation.html#using-rackspace">Rackspace</a></li>
-					<li><a href="https://docs.clusterhq.com/en/latest/gettingstarted/installation.html#using-digitalocean">Digital Ocean</a></li>
-					<li><a href="https://docs.clusterhq.com/en/latest/gettingstarted/installation.html#vagrant">Vagrant Machine</a></li>
+					<!-- https://github.com/sphinx-doc/sphinx/issues/1702 -->
+					<li><a href="{{ pathto('indepth/installation') }}#aws-install">AWS</a></li>
+					<li><a href="{{ pathto('indepth/installation') }}#rackspace-install">Rackspace</a></li>
+					<li><a href="{{ pathto('indepth/installation') }}#digitalocean-install">Digital Ocean</a></li>
+					<li><a href="{{ pathto('indepth/installation') }}#vagrant-install">Vagrant Machine</a></li>
```

Comment:
@msh100 @dwgebler These changes (`s/gettingstarted/indepth/` and using ``#X-install``) probably want to go on the main site as well.